crypto/crypto.go

```diff
@@ -280,7 +280,27 @@ func PubkeyToAddress(p ecdsa.PublicKey, nodeLocation common.Location) common.Add
 }
 
 func PubkeyBytesToAddress(pub []byte, nodeLocation common.Location) common.Address {
```

Comment:
Ah, I misunderstood where you were making the change. This function should accept uncompressed keys still.

Internally, we will still operate on uncompressed keys. It is only requests inbound on the RPC that will include compressed keys, so instead, decompress keys at the RPC before they get handled.